### ISSUE TITLE
downloading images directly freezes while waiting and no progress bar or size

### ISSUE THREAD

**Issue by NijazM** [gnosygnu/xowa#776]
I noticed that when I download some larger image or adudio or video, by clicking thumbnail on wikipedia english, then it shows progress bar only for a second and then freezes until file downloads and opens in some image/media viewer app. For example original photos from commons. Actually there is no any progress or file size, unlike in thumbnails or wikipedia articles.

**Comment by gnosygnu** [comment]
Yeah, this looks like an issue. Marking it as a bug, but it may take a while for me to get to...

**Comment by NijazM** [comment]
Also in server images don't even download, but error is reported.
